Config.cmake.in

```diff
@@ -0,0 +1,11 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/blmc_controllersTargets.cmake")
+
+include(CMakeFindDependencyMacro)
+
+# we do not add the other dependencies because these are header files lib
+find_dependency(yaml-cpp CONFIG REQUIRED)
+find_dependency(Eigen3 REQUIRED)
```

Comment:
wrong list, all these have target that needs to be found by the child project (basically all but the mpi_cmake_module):
```
find_dependency(pybind11 CONFIG REQUIRED)
find_dependency(Eigen3 REQUIRED)
find_dependency(pinocchio REQUIRED)
find_dependency(eiquadprog REQUIRED)
find_dependency(Boost REQUIRED COMPONENT python) # has a target inside that you need
find_dependency(yaml_utils REQUIRED) # has a target inside that you need
```

Reply:
I've updated the `Config.cmake.in` based on your suggestion. For now I am using `SET(BOOST_COMPONENTS python)` still though as specifing the component on the find_dependency is not working (see other comment).